### ISSUE TITLE
Faction Camp recognizes activated Integrated toolset bionic as cooking tool, when assigning cooking task

### ISSUE THREAD

**Issue by DamienRoyan** [CleverRaven/Cataclysm-DDA#58676]
### Describe the bug

It is an exploit, where as long as the player has their Integrated toolset bionic active, it can be used for the various cooking tasks in the faction bulletin board.
This means the player can avoid the expense for getting a working oven or hotplate with enough charges.

This also clashes with the realism aspect of the game, as I can assign the mission and then piss off into the sunset. How does my NPC continue to cook if I took the toolset bionic with me?

### Steps to reproduce

1. Acquire a faction camp with appropriate advancements and an Integrated Toolset bionic. Have an oven or hotplate on site so the prompt for which tool to use comes up.
2. Select any one task and the prompt for which tool to use will show. Integrated Toolset will appear

### Expected behavior

It should not recognize this as a tool as it expects the player to be on site.
It shouldn't be uncommon that a player assigns tasks like auto-cook before leaving on a quick trip to a lumber/steel mill, NPC faction location or clearing out a small segment of the city.

### Screenshots

![image](https://user-images.githubusercontent.com/88375697/175536965-57b115c9-e028-4323-a963-f2b0baeef636.png)
Note that Integrated Toolset does not appear in this window ^ in the list

![image](https://user-images.githubusercontent.com/88375697/175536902-1c8b1120-cdd8-4b65-ac85-f80b8f517d2e.png)

### Versions and configuration

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on unknown Windows version
- Game Version: 131cea4 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills]
]

**Comment by catdach** [comment]
looks like a dupe of #55229

**Comment by PatrikLundell** [comment]
Yes, it's a duplicate, and yes, bionic power ought to be available only for PC crafting (although it gets tricky when companion assisted batch crafting enters the picture).

**Comment by DamienRoyan** [comment]
> Yes, it's a duplicate, and yes, bionic power ought to be available only for PC crafting (although it gets tricky when companion assisted batch crafting enters the picture).

Companion assisted batch crafting isn't an issue as the player must also be present.

Who's to say that it's not either the player is having the npc hold something in place while they're welding/soldering it, or the NPC using the welder/soldering iron on some pieces the player is holding exactly how they want it

**Comment by PatrikLundell** [comment]
In some cases it makes sense for the PC to do the powered stuff with the assistance from a companion, but on others it doesn't really do that (10 companions assisting in something that's almost exclusively a powered task, for instance). That's why it gets tricky.

**Comment by hexagonrecursion** [comment]
/confirmed

> Integrated toolset

I assume you mean "Integrated Multitool" (id: bio_tools)

![image](https://user-images.githubusercontent.com/52621858/177081190-a498de0b-8b57-4c1b-aff6-0522b8a1b8a5.png)


- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 36 (Thirty Six); Release: 36; Codename: ThirtySix; 
- Game Version: cccb059 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by DamienRoyan** [comment]
> /confirmed
> 
> > Integrated toolset
> 
> I assume you mean "Integrated Multitool" (id: bio_tools)

Indeed
